### DISCUSSION TITLE
Any easy to follow tutorials regarding installing extensions for AUTOMATIC1111 Docker image?

### DISCUSSION THREAD

**Discussion by Valachio** [AbdBarho/stable-diffusion-webui-docker#639] [Q&A]
I am trying to use the AUTOMATIC1111 Docker image, and I need to install several extensions. I'm a noob to Docker and it's a struggle for me to understand the code. 

The only resource I can find about regarding installing extensions is [this issue](https://github.com/AbdBarho/stable-diffusion-webui-docker/issues/172) but the advice is scattered.

Is there a guide out there that can guide me on installing extensions for AUTOMATIC1111 on this Docker image?

**Comment by DevilaN** [comment]
Hello.
You install them as usual in A1111 most of the time (using extensions tab). Extensions have their own `requirements.txt` file which is being respected and required PYTHON dependencies are being installed.
There are extensions that require more things to do (like installing external software), but right now we do not worry about it.

Is there any particular extension you have problems with? Have you managed to install any extension successfully?
